app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import Menubar from "components/Common/Menubar";
+import Table from "components/Dashboard/Contacts/Table";
+
+import { CONTACTS_DATA, MENUBAR_DATA } from "./constants";
+
+function Contacts() {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [, setCategory] = useState("All");
+  const [, setShowNewContactPane] = useState(false);
```

Comment:
Not sure I understand the need to make this a state if we aren't actually utilising this as a state. Why are these declared as states?

Reply:
showNewContactPane is passed in CreatePane component which is added in next pull request.
Incase of category I forgot to add it to the menubar title. I have created a new pull request with that change.
https://github.com/soorajjtk/neeto-ui-challenge/pull/12